cypress/integration/test-helper.js

```diff
@@ -16,9 +16,14 @@ const shouldContainStyles = (element, css, styling) => {
   });
 };
 
+const shouldHaveMetadata = (name, content) => {
+  cy.get(`head meta[name=${name}]`).should('have.attr', 'content', content);
```

Comment:
You should use [this helper function](https://github.com/bbc/simorgh/pull/157/files#diff-6792feee673d43eeee989cd644d01bc5R7) for getting elements.